crates/amaru-ledger/src/governance/ratification/proposals_forest.rs

```diff
@@ -954,13 +1104,15 @@ mod tests {
                 );
 
                 // Control that the enacted proposal is one of the new root.
-                prop_assert!(
-                    forest.roots().protocol_parameters.as_deref() == Some(id.as_ref()) ||
-                    forest.roots().hard_fork.as_deref() == Some(id.as_ref()) ||
-                    forest.roots().constitutional_committee.as_deref() == Some(id.as_ref()) ||
-                    forest.roots().constitution.as_deref() == Some(id.as_ref()),
-                    "none of the roots match the just enacted proposal",
-                );
+                if !matches!(proposal, ProposalEnum::Orphan(..)) {
```

Comment:
this `!matches!` looks weird at first sight. Although they are not needed, perhaps parens would help here?

Reply:
cargo-fmt gets rid of them. I've been using helpers like `is_orphan()` in other places, I could do the same here 👍